03/app.js

```diff
@@ -0,0 +1,18 @@
+const btnList = document.querySelectorAll('button')
+console.log(btnList)
+
+const changeTxt = function()
+{
+    this.innerText = "clicked";
+    console.log('clicked')
+    btnList.forEach(function(el)
```

Comment:
tutaj wykonujesz działania dla wszystkich, a masz tylko dla tego klikniętego.
Możesz użyć `this` lub `e.target`